src/index.js

```diff
@@ -1,32 +1,76 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable prefer-destructuring */
+
```

Comment:
- [ ] I think it's not a good practise to disable linters, kindly resolve the issues that these linters pointed out. 👍